lib/lookup/src/lookup_v2/mod.rs

```diff
@@ -21,47 +21,72 @@ macro_rules! path {
     }};
 }
 
+/// Syntactic sugar for creating a pre-parsed path.
+/// This path points at an event (as opposed to metadata).
+#[macro_export]
+macro_rules! event_path {
+    ($($segment:expr),*) => {{
+           ($crate::lookup_v2::PathPrefix::Event, &[$($crate::lookup_v2::BorrowedSegment::from($segment),)*])
+    }};
+}
+
+/// Syntactic sugar for creating a pre-parsed path.
+/// This path points at metadata (as opposed to the event).
+#[macro_export]
+macro_rules! metadata_path {
+    ($($segment:expr),*) => {{
+           ($crate::lookup_v2::PathPrefix::Metadata, &[$($crate::lookup_v2::BorrowedSegment::from($segment),)*])
+    }};
+}
+
 /// Syntactic sugar for creating a pre-parsed owned path.
 ///
 /// This allocates and will be slower than using `path!`. Prefer that when possible.
 /// The return value must be borrowed to get a value that implements `Path`.
 ///
 /// Example: `owned_path!("foo", 4, "bar")` is the pre-parsed path of `foo[4].bar`
 #[macro_export]
-macro_rules! owned_path {
+macro_rules! owned_value_path {
```

Comment:
There seem to be changes that have not been reflected in their respective comments. Might be worth double checking for the other renames as well